srcpkgs/libfm-qt/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'libfm-qt'
 pkgname=libfm-qt
```

Comment:
Please add notices to both packages to keep them in sync similar how it's done for example in vala and valadoc